### PR TITLE
Add voicelines SCP-3114

### PR DESCRIPTION
**Pull request by Rysik5318**

### CODE REVIEW

File: Exiled.Events/EventArgs/Scp3114/VoiceLinesEventArgs.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+{
+    using API.Features;
+    using Exiled.API.Features.Roles;
+    using Interfaces;
+    using static PlayerRoles.PlayableScps.Scp3114.Scp3114VoiceLines;
+
+    /// <summary>
+    ///     Contains all information prior to sending voiceline SCP-3114.
+    /// </summary>
+    public class VoiceLinesEventArgs : IScp3114Event
```
Comment (IRacle1) [inline Exiled.Events/EventArgs/Scp3114/VoiceLinesEventArgs.cs:18]: also should inherit `IDeniableEvent`

File: Exiled.Events/Patches/Events/Scp3114/VoiceLines.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    using PlayerRoles.PlayableScps.Scp3114;
+    using static NineTailedFoxAnnouncer;
+    using static PlayerRoles.PlayableScps.Scp3114.Scp3114VoiceLines;
+    using UnityEngine;
+
+    /// <summary>
+    ///     Patches <see cref="Scp3114Disguise.OnProgressSet()" />.
+    ///     Adds the <see cref="Handlers.Scp3114.TryUseBody" /> event.
+    /// </summary>
+    [EventPatch(typeof(Scp3114), nameof(Scp3114.TryUseBody))]
```
Comment (IRacle1) [inline Exiled.Events/Patches/Events/Scp3114/VoiceLines.cs:25]: Fix those `EventPatch` attribute and docs

**Review by IRacle1 — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Player.cs [hunk 3]
```diff
@@ -1839,6 +1839,23 @@ public void Broadcast(Broadcast broadcast, bool shouldClearPrevious = false)
                 Broadcast(broadcast.Duration, broadcast.Content, broadcast.Type, shouldClearPrevious);
         }
 
+        /// <summary>
+        /// Drops an item from the player's inventory.
+        /// </summary>
+        /// <param name="item">The item to be dropped.</param>
+        public void DropItem(Item item, bool isThrown = false) => Inventory.UserCode_CmdDropItem__UInt16__Boolean(item.Serial, isThrown);
```
[the remark this answers is not in the document]
Comment (louis1706) [reply Exiled.API/Features/Player.cs:1846]: Okay just see a Null Check Needed for item

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Player.cs [hunk 4]
```diff
@@ -1839,6 +1839,19 @@ public void Broadcast(Broadcast broadcast, bool shouldClearPrevious = false)
...
```
Comment (louis1706) [inline Exiled.API/Features/Player.cs:1846]: You missed the null check for item

**Review by louis1706 — CHANGES_REQUESTED** [review]

**Review by louis1706 — APPROVED** [review]

**Comment by ALEXWARELLC** [comment]
👍

**Pull request closed without merging**